### ISSUE TITLE
How to add dropout and reguralizer to the unet_2d model

### ISSUE THREAD

**Issue by AnjuMKaimal** [yingkaisha/keras-unet-collection#59]
I am trying to use the unet_2d model from collection . However the curve shows that data is overfitting. I want to add regularizer and droput to the layers . How can I incorporate this ?
